fastlane/lib/fastlane/actions/docs/frame_screenshots.md

```diff
@@ -2,7 +2,7 @@
   <img src="/img/actions/frameit.png" width="250">
 </p>
 
-###### Quickly put your screenshots into the right device frames
+###### Easily put your screenshots into the right device frames
```

Comment:
I think that @KrauseFx should decide if it's ok to change this tagline.

Reply:
This actually was just a change @funnel20 made to start the PR, can just be reverted.

Reply:
Please revert. The whole idea of this PR is to bring this docu in line with the current code implementation and this word is not related.